setup.py

```diff
@@ -13,6 +14,7 @@
 
 requirements = [
     "Click>=7.0",
+    "click-loguru==1.3.6",
```

Comment:
Do you really want to specify an exact version. Are you sure this will still work in a year from now. I see the risk that we can't resolve this hardcoded dependency anymore at some point in the future.

Reply:
Unless PyPI goes offline, this should always be available. I would even prefer exact versions to non-specific ones: this way I know exactly what I get and (while unlikely) it might be that getting whatever is "newest" has slightly different behaviors. Finding that out after the fact sounds painful. For a small dependency like this it isn't a very big problem, but imagine for instance if the YAML loading library suddenly behaves differently.

Reply:
Ok, thanks for clarifying. I had problems in another project (old python 2.x project) where hardcoded versions started to make problems when python 2.7 started to die...